src/session/cookie-store/index.ts

```diff
@@ -25,12 +25,17 @@ export default class CookieSessionStore implements ISessionStore {
     const { cookieSecret, cookieName } = this.settings;
 
     const cookies = parseCookies(req);
-    const cookie = cookies[cookieName];
-    if (!cookie || cookie.length === 0) {
-      return null;
+    const cookieCount = cookies[`${cookieName}.c`] || 1;
```

Comment:
Any thoughts on making this backwards compatible? So instead of defaulting to `1` it uses `const cookie = cookies[cookieName];` as the cookie when there is no cookie with the count present. That way existing sessions would be valid even after the library upgrade.

Reply:
A had this on my mind, but on the other hand it would complicate the code quite a bit. And at the end everyone only has to restart their sessions maximum once after switching to the new version.

Reply:
@mrbig which could be unacceptable for some users.
It would need to be either backwards compatible or a breaking change.

Reply:
Ok, added some backward compatibility code and tests